src/secure-proxy-handler.ts

```diff
@@ -67,7 +66,9 @@ function getPropertyDescriptor(o: any, p: PropertyKey): PropertyDescriptor | und
 
 function copySecureOwnDescriptors(env: SecureEnvironment, shadowTarget: SecureShadowTarget, target: SecureProxyTarget) {
     // TODO: typescript definition for getOwnPropertyDescriptors is wrong, it should include symbols
-    const descriptors = getOwnPropertyDescriptors(target);
+    const descriptors: PropertyDescriptorMap = callWithErrorBoundaryProtection(env, () => {
+        return getOwnPropertyDescriptors(target);
+    });
```

Comment:
What error is this guarding? ☝️

Reply:
same as above, proxy traps throwing in user-land in the sandbox.

Reply:
Nice, I hadn't thought about the trap case.